From 39aa947784142babe9d1618ef0d7189d062a0e58 Mon Sep 17 00:00:00 2001
From: Matthew Brett <matthew.brett@gmail.com>
Date: Mon, 27 Jul 2020 18:24:08 +0100
Subject: [PATCH] BF: fix tests for \x00 appearing in doctree

Docutils 0.16 appears to insert \x00 characters into the doctree for an
escaped dollar, which then appear in our simple text representation.

Strip these.
---
 texext/tests/test_tinypages.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/texext/tests/test_tinypages.py b/texext/tests/test_tinypages.py
index 7316995..156e4b4 100644
--- a/texext/tests/test_tinypages.py
+++ b/texext/tests/test_tinypages.py
@@ -35,6 +35,8 @@ def test_some_math(self):
         doctree = self.get_doctree('some_math')
         assert len(doctree.document) == 1
         tree_str = self.doctree2str(doctree)
+        # Strip 0s that appear from docutils 0.16
+        tree_str = tree_str.replace('\x00', '')
         if SPHINX_ge_1p5:
             back_ref = (
                 '<paragraph>Refers to equation at '
